src/Scripting/Core/ScriptOptions.cs

```diff
@@ -54,6 +53,7 @@ private static ImmutableArray<MetadataReference> GetDefaultMetadataReferences()
                 "System.Reflection.Extensions",
                 "System.Reflection.Primitives",
                 "System.Runtime",
+                "System.Runtime.Extensions",
```

Comment:
Let's add System.Runtime.Extensions here and to the csi.rsp

---
In reply to: [110983277](https://github.com/dotnet/roslyn/pull/18471#discussion_r110983277) [](ancestors = 110983277,110982988,110977656,110973145,110970347)